src/test/java/org/prebid/server/validation/RequestValidatorTest.java

```diff
@@ -289,6 +292,138 @@ public void validateShouldReturnValidationMessageWhenAudioAttributePresentButAud
                 .containsOnly("request.imp[0].audio.mimes must contain at least one supported MIME type");
     }
 
+    @Test
```

Comment:
Please add a test for the case when `banner.w` and `banner.h` are present, but `banner.format == null`.